### PR TITLE
[10.0][NEW] pos_menu, pos_multi_session_menu

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_menu/static/src/js/pos.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    models.load_models([{
+        model: 'pos.tag',
+        field: [],
+        domain: null,
+        loaded: function(self, tags) {
+            self.tags = [];
+            // save tags of current POS
+            tags.forEach(function(tag){
+                if ($.inArray(self.config.id, tag.pos_ids) !== -1) {
```
Comment (yelizariev) [inline pos_menu/static/src/js/pos.js]: Why don't you filter tags out in ``domain``?

File: pos_menu/static/src/js/pos.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                    self.tags.push(tag);
+                }
+            });
+            // add new domain for product
+            self.models.forEach(function(model) {
+                if (model.model === "product.product") {
+                    model.domain.push(['tag_ids', 'in', self.config.tag_ids]);
```
Comment (yelizariev) [inline pos_menu/static/src/js/pos.js]: :-1: 
When tag list is empty, it must load all products

File: pos_menu/static/src/js/pos.js (hunk 1 above)
Comment (GabbasovDinar) [reply pos_menu/static/src/js/pos.js]: @yelizariev it's not a required code. I save the tags of the current POS for future use. What do you mean "filter tags out in domain"? How can I do it?

File: pos_menu/static/src/js/pos.js (hunk 2 above)
Comment (GabbasovDinar) [reply pos_menu/static/src/js/pos.js]: @yelizariev not problem)

File: pos_menu/static/src/js/pos.js (hunk 1 above)
Comment (GabbasovDinar) [reply pos_menu/static/src/js/pos.js]: @yelizariev now I understand what you mean. I need use '''domain''', no problem)

**Comment by GabbasovDinar** [comment]
@ilmir-k @yelizariev  Can you check the module?

**Comment by GabbasovDinar** [comment]
@yelizariev Can you review the module?

**Pull request merged**